src/backend/requirements.txt

```diff
@@ -1,15 +1,16 @@
 black==22.3.0
-boto3==1.22.4
-django-extensions==3.1.5
+boto3==1.37.32
+django==5.2
```

Comment:
Unclear why Django was not added as a dep in the requirements before. But now added here